README.md

```diff
@@ -1,190 +1,145 @@
 ```bash
 export path_model=https://github.com/SAP/credential-digger/releases/download/PM-v1.0.1/path_model-1.0.1.tar.gz
+export snippet_model=https://github.com/SAP/credential-digger/releases/download/SM-v1.0.0/snippet_model-1.0.0.tar.gz
+```
+```bash
 python -m credentialdigger download path_model
+python -m credentialdigger download snippet_model
 ```
 
-### Code Snippet Model
-
-The code Snippet model identifies the password based authentication in a code and differeciate between real and fake passwords. A pre-trained Code Snippet Model [is available here](https://github.com/SAP/credential-digger/releases/download/SM-v1.0.0/snippet_model-1.0.0.tar.gz)
+> **WARNING**: If you build the code from scratch (i.e., you don't install the client via
+pip), don't run the download command from the installation folder of
+_credentialdigger_ in order to avoid errors in linking.
 
-WARNING: This Model is pre-trained with synthetic data in order to protect privacy. It helps to reduce the False Positives related to password recongnition, but with a lower precision compared to a Model pre-trained with real data.
+> **WARNING**: We provide the pre-trained models, but we do not guarantee the efficiency of these models. If you want more accuracte machine learning models, you can train your own models (just replace the binaries by your own models) or use the fine-tuning option.
 
-### Integrate ML Models in the UI
 
-**ML models are not supported yet in the UI** "out of the box", but they can be
-enabled with a little effort.
+## Usage
 
-To use ML models in the docker container running the ui, set their address in
-the `.env` file (the pre-trained Path and Snippet Models are already set).
-If the address is correct, the model will be installed at container building
-time.
-However, the current server is not executing any model during the scan of a
-repository. To enable a model, it must be listed as an argument in the `scan`
-function (as would be done in the client) in the `ui/server.py` file.
+To instanciate a client, connected to the database :
 
+```python
+from credentialdigger.cli import Client
+c = Client(dbname='MYDB', dbuser='POSTGRES_USER',
+						  dbpassword='POSTGRES_PASSWORD',
+						  dbhost='localhost', dbport=5432)
+```
+### Scan a repository
 
-## Usage (client)
+You first need to add the repository to the database. Then you can proceed to a scan :
 
 ```python
-from credentialdigger.cli import Client
-c = Client(dbname='MYDB', dbuser='MYUSER', dbpassword='*****',
-           dbhost='localhost', dbport=5432)
+c.add_repo(url='https://github.com/user/repo')
+new_discoveries = c.scan(repo_url='https://github.com/user/repo',
+                         category=None,
+                         scanner=GitScanner,
+                         models=['PathModel', 'SnippetModel'],
+                         exclude=None,
+                         force=True,
+                         generate_snippet_extractor=False,
+                         verbose=True)
```

Comment:
Don't show all the parameters, but only the "most useful" ones, i.e., the repo url, the models, and verbose

Reply:
Ok, I'll modify it